lib/wallaby/feature.ex

```diff
@@ -0,0 +1,220 @@
+defmodule Wallaby.Feature do
+  @includes_ecto Code.ensure_loaded?(Ecto.Adapters.SQL.Sandbox) &&
+                   Code.ensure_loaded?(Phoenix.Ecto.SQL.Sandbox)
```

Comment:
What do you think about moving this below the `@moduledoc`? Generally I've seen moduledoc as the first thing and instance variables below.

Reply:
Will do 👍